### ISSUE TITLE
Update lager config to not use deprecated features

### ISSUE THREAD

**Issue by larshesel** [vernemq/vernemq#1209]
### Environment

- VerneMQ Version: 1.8.0
- OS: linux
- Erlang/OTP version (if building from source):
- VerneMQ configuration (vernemq.conf) or the changes from the default
- Cluster size/standalone: k8s cluster vernemq_only: https://github.com/vernemq/vmq-operator/tree/master/examples/only_vernemq

Just noticed this in the logs:

```
 WARNING: This is a deprecated console configuration. Please use "[{level,info}]" instead. 
```

I suppose this is something we missed when upgrading lager and we need to update the configuration.

**Comment by codeadict** [comment]
This one could be closed now

**Comment by larshesel** [comment]
closed via #1211
